packages/react-components/react-persona/package.json

```diff
@@ -45,11 +45,9 @@
     "react-dom": ">=16.8.0 <19.0.0"
   },
   "beachball": {
-    "tag": "beta",
     "disallowedChangeTypes": [
       "major",
-      "minor",
-      "patch"
+      "prerelease"
```

Comment:
This seems to be causing an issue with the [change files check](https://github.com/microsoft/fluentui/actions/runs/3831480857/jobs/6520660396).

Reply:
yeah :/ there's a change file that was just recently added that uses `type: prerelease` so it's conflicting. I'm checking what to do about it :(

Reply:
You may have to wait till the new version of fluent gets released and that change file gets incorporated into it